dipy/reconst/csdeconv.py

```diff
@@ -150,15 +150,15 @@ def __init__(self, S0, dwi_response, bvalue=None):
         self.S0 = S0
         self.dwi_response = dwi_response
         self.bvalue = bvalue
-        self.m = np.zeros(len(dwi_response))
-        self.sh_order = 2 * (len(dwi_response) - 1)
-        self.n = np.arange(0, self.sh_order + 1, 2)
+        self.m_value = np.zeros(len(dwi_response))
+        self.sh_order_max = 2 * (len(dwi_response) - 1)
+        self.l_value = np.arange(0, self.sh_order_max + 1, 2)
```

Comment:
`self.l_value` and `self.m_value` are arrays here, shouldn't they have `s` at the end?